website/docs/utilities/popover-primitive/partials/code/how-to-use.md

```diff
@@ -0,0 +1,133 @@
+!!! Warning
+
+This component is intended only for internal Helios use. If you need to use it, please [contact](/about/support) the Design Systems Team.
+!!!
+
+## How to use this component
+
+The `PopoverPrimitive` is a **headless component** that associates a "toggle" element with a "popover" element (both elements act as containers). "Soft" (hover/focus) or "click" event listeners can be assigned to the "toggle", and when triggered they toggle the visibility of the "popover".
+
+When the "popover" is visible, it can be closed in various ways: toggling via the "soft" or "click" events, clicking outside of the popover, or via the `esc` key.
+
+Under the hood the component uses the [native web Popover API](https://developer.mozilla.org/en-US/docs/Web/API/Popover_API) to promote the popover content to the [top layer](https://developer.mozilla.org/en-US/docs/Glossary/Top_layer). This solves the issues related to stacking contexts and provides the "light dismiss" (click outside / `esc` key) out of the box.
+
+The primitive also uses the third-party library [Floating UI](https://floating-ui.com/) to provide the anchoring and automatic positioning/collision detection functionality.
+
+For older browsers (in particular Firefox 124 and older) that don't support the Popover API, it uses a [Popover Polyfill](https://github.com/oddbird/popover-polyfill) library to emulate the native behaviour.
+
+!!! Insight
+
+**Learn more**
+
+- For details about the native web Popover API see: [MDN / Popover API](https://developer.mozilla.org/en-US/docs/Web/API/Popover_API)
+- For details about the Floating UI third-party library see: [Floating UI](https://floating-ui.com/)
+
+For details about the Floating UI third-party library see: [Floating UI](https://floating-ui.com/)
```

Comment:
```suggestion
```

Reply:
Duplicated

Reply:
Probably effect of some auto-resolution of conflicts.